docs/source/legacy.rst

```diff
@@ -7,6 +7,30 @@ consistent code base.
 
 This section lists the major breaking changes introduced.
 
+Breaking changes in Perceval 0.13
+---------------------------------
+
+Source parameter has been removed from Processor and Converter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The line of ^^^^^^^^^.... is too long and will produce a warning during doc generation